webproject/src/main/java/com/google/sps/data/Order.java

```diff
@@ -0,0 +1,60 @@
+package com.google.sps.data;
+
+public class Order {
+    private String user;
+    private String product;
```

Comment:
are you putting the order as the child of room in the firebase?
this may have several cons:
1. order may not have id, like roomId
2. we may have a hard time to query "my orders" if we are going to implement it in the future.

I would suggest to have a separate table for order, and put roomId as a attribute in the Order

Reply:
similarly for users. you can create a table called RoomParticipant which only contains roomId and userId

Reply:
Ok! Have changed orders to a separate document in Firebase rather than a child of room. For users, I'll change it on the MyOrders branch!